var/spack/repos/builtin/packages/libflame/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libflame(AutotoolsPackage):
+    """libflame is a portable library for dense matrix computations,
+    providing much of the functionality present in LAPACK, developed
+    by current and former members of the Science of High-Performance
+    Computing (SHPC) group in the Institute for Computational
+    Engineering and Sciences at The University of Texas at Austin.
+    libflame includes a compatibility layer, lapack2flame, which
+    includes a complete LAPACK implementation."""
+
+    homepage = "https://www.cs.utexas.edu/~flame/web/libFLAME.html"
+    url      = "https://github.com/flame/libflame/archive/5.1.0.tar.gz"
+    git      = "https://github.com/flame/libflame.git"
+
+    version('develop', branch='master')
```

Comment:
You can just call this version `master` if you want.

Reply:
Isn't the convention in spack develop>[numeric versions]>all other string versions ? Since the last release was so long ago, I'd prefer to keep it as develop unless I'm missing sth.

Reply:
That was the convention, until #1983 was merged. Now develop > master > numeric versions > all other string versions.

Reply:
Great! , it used to cause a lot confusion outside spack as to why is spack calling a version develop when the package developers call it master.